backend/api/projects.py

```diff
@@ -368,48 +372,46 @@ def update_comment(project_id, comment_id):
 @projects.route('/api/projects/<uuid:project_id>/comment/<uuid:comment_id>', methods=['DELETE'])
 @login_required
 def delete_comment(project_id, comment_id):
-    """Deletes comment
+    """Deletes comment by comment_id
 
     Args:
         project_id, comment_id
 
     Returns:
         response: Success response
                   or 404 if project is not found
-                  or 400 if validation error occurs
+                  or 500 if validation error occurs
```

Comment:
this is the DELETE call so I was mistaken in the first place, 500 is correct and PUT and POST were already returning correct codes